src/App.tsx

```diff
@@ -1,11 +1,85 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
+import { getMovie } from './api';
+import { MovieData } from './types/MovieData';
+import { ResponseError } from './types/ReponseError';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [query, setQuery] = useState<string>('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [notFoundMovie, setNotFoundMovie] = useState(false);
+  const [finished, setFinished] = useState(true);
+  const [showErrorMessage, setShowErrorMessage] = useState(false);
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
```suggestion
  const [isErrorMessage, setIsErrorMessage] = useState(false);
```